app/scripts/modules/core/src/notification/selector/types/bearychat/BearychatNotificationType.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { FormikFormField, TextInput, Validators } from 'core/presentation';
+import { INotificationTypeCustomConfig } from 'core/domain';
+
+export class BearychatNoficationType extends React.Component<INotificationTypeCustomConfig> {
+  public render() {
+    const { fieldName } = this.props;
+    return (
+      <div className="sp-margin-m-bottom">
+        <div className={'form-group'}>
+          <label className={'col-md-4 sm-label-right'}>Email Address</label>
```

Comment:
Would prefer to use `<FormikFormField label="Email Address"` in the future, but this works for now.

Reply:
I know I will refactor it after.